rest-client/pom.xml

```diff
@@ -154,6 +154,30 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Are these added into the JAR? That's cool, I never knew how to do this. Also is there some way to preserve line numbers and such so debuggers work correctly? I think you have to do something to have that in the JAR as well. (Not in this PR, just asking).